scripts/concurrent_test.sh

```
@@ -0,0 +1,321 @@
+echo "Testing $CONCURRENT_REQUESTS concurrent requests..."
+echo
+
+# Create results directory
+mkdir -p "$RESULTS_DIR"
+
+
+# Function to send a single request
+send_request() {
+    local request_id="$1"
+    local result_file="$RESULTS_DIR/result_$request_id.txt"
+    local start_time=$(date +%s.%N)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)